src/util/BaseLoginRouter.js

```diff
@@ -206,6 +207,13 @@ function (Okta, BrowserFeatures, Settings,
           .done();
       }
 
+      if (this.settings.get('colors.brand') && !ColorsUtil.isLoaded()) {
```

Comment:
Should we add a comment here stating that the intended use of this functionality is that it **only runs once**?